internal/controller/apiextensions/usage/reconciler.go

```diff
@@ -406,11 +406,16 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	}
 
 	u.Status.SetConditions(xpv1.Available())
-	r.record.Event(u, event.Normal(reasonUsageConfigured, "Usage configured successfully."))
+
 	// We are only watching the Usage itself but not using or used resources.
 	// So, we need to reconcile the Usage periodically to check if the using
 	// or used resources are still there.
-	return reconcile.Result{RequeueAfter: r.pollInterval}, errors.Wrap(r.client.Status().Update(ctx, u), errUpdateStatus)
+	if !reflect.DeepEqual(u, orig) {
```

Comment:
Nit: Slight preference for `cmp.Equal` here: https://github.com/google/go-cmp

It probably won't make a difference in this specific use case, but I think we mostly use `cmp.Equal` in this codebase. It gives us more configurability on what equality means if/when we need it.

Reply:
updated.